lib/dor/config.rb

```diff
@@ -55,8 +55,10 @@ def make_rest_client(url, cert = Config.ssl.cert_file, key = Config.ssl.key_file
     end
 
     def make_solr_connection(add_opts = {})
-      opts = Config.solrizer.opts.merge(add_opts).merge(
-        :url => Config.solrizer.url
+      c = Dor::Config.solr || Dor::Config.solrizer
```

Comment:
might be nice to leave a comment somewhere about supporting the deprecated config namespace, and/or give users of it a deprecation warning.